crypto/pkcs7/pkcs7_test.cc

```diff
@@ -1673,7 +1673,7 @@ TEST(PKCS7Test, DataInitFinal) {
   bio_in.reset(BIO_new(BIO_s_mem()));
   ASSERT_TRUE(p7);
   ASSERT_TRUE(PKCS7_set_type(p7.get(), NID_pkcs7_encrypted));
-  bio.reset(PKCS7_dataInit(p7.get(), bio_in.get()));
+  bio.reset(PKCS7_dataInit(p7.get(), bio_in.release()));
```

Comment:
Actually, I think `PKCS7_dataInit` only takes ownership on a successful return.  Since this is expected to be a failed call, the original was likely correct.